examples/gatsby/sample-gatsby-site/gatsby-browser.js

```
@@ -0,0 +1,46 @@
+// Initialize rudderanalytics buffer
+export const onClientEntry = () => {
+  if (typeof window !== 'undefined') {
+    (function() {
+      "use strict";
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)